Add label filter: "Show starred events" --> internal server error

**Describe the bug**
In a sketch with stared events, search for ```*``` and then click on ```add label filter --> Show starred events```. This will generate an ```Internal server error``` and add a filter without any text (see screenshot).

Workaround is to use either the ```Tags --> Starred``` Suggestion in the new explore view (see screenshot 2)

**To Reproduce**
In a sketch with stared events, search for ```*``` and then click on ```add label filter --> Show starred events```. This will generate an ```Internal server error``` and add a filter without any text (see screenshot).

**Expected behavior**
Add a search filter for the label: __ts_star

**Screenshots**
![Screenshot 2021-07-09 at 09 25 23](https://user-images.githubusercontent.com/741037/125041497-1ca31500-e099-11eb-88e2-a1b4637b46a0.png)

![Screenshot 2021-07-09 at 09 31 45](https://user-images.githubusercontent.com/741037/125041539-29276d80-e099-11eb-87e3-c77ca6d334df.png)

**Additional context**
Could not reproduce on demo.timesketch.org but locally with latest version, I guess because we did not deploy the latest version to demo server.

(found by @Onager )

Stracktrace: 
```
[2021-07-09 07:41:46 +0000] [697] [ERROR] Error handling request /api/v1/sketches/1/explore/
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/usr/local/lib/python3.8/dist-packages/gunicorn/workers/sync.py", line 176, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2463, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2449, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_restful/__init__.py", line 269, in error_router
    return original_handler(e)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1866, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.8/dist-packages/flask/_compat.py", line 38, in reraise
    raise value.with_traceback(tb)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_restful/__init__.py", line 269, in error_router
    return original_handler(e)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.8/dist-packages/flask/_compat.py", line 38, in reraise
    raise value.with_traceback(tb)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python3.8/dist-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/usr/local/src/timesketch/timesketch/api/v1/resources/explore.py", line 224, in post
    result = self.datastore.search(
  File "/usr/local/src/timesketch/timesketch/lib/datastores/elastic.py", line 520, in search
    query_dsl = self.build_query(
  File "/usr/local/src/timesketch/timesketch/lib/datastores/elastic.py", line 366, in build_query
    labels.append(chip['value'])
KeyError: 'value'

```

--- a/timesketch/frontend/src/views/Explore.vue
+++ b/timesketch/frontend/src/views/Explore.vue
@@ -949,7 +949,7 @@ export default {
       this.selectedLabels.forEach(label => {
         let chip = {
           field: '',
-          value: label.label,
+          value: label,
           type: 'label',
           operator: 'must',
           active: true,

